src/emulator/controller.ts

```diff
@@ -132,11 +132,36 @@ export async function startAll(options: any): Promise<void> {
       options.config.get("functions.source")
     );
 
+    let inspectFunctions = undefined;
+
+    // If the flag is provided without a value, use the Node.js default
+    if (options.inspectFunctions === true) {
+      options.inspectFunctions = "9299";
+    }
+
+    if (options.inspectFunctions) {
+      inspectFunctions = parseInt(options.inspectFunctions, 10);
```

Comment:
nit: I bet `10bottlesofwine` will parse as `10`. Is this expected? If not, try `Number(x)`.

Reply:
Thanks!